src/ClientInterface.php

```diff
@@ -8,16 +8,6 @@
 
 interface ClientInterface
 {
-    /**
-     * @return ContainerInterface
-     */
-    public function getContainer(): ContainerInterface;
-
-    /**
-     * @return mixed
-     */
-    public function getFromContainer(string $id);
```

Comment:
This is only used in two clients and from I've gathered it can easily updated after this fix :+1: